src/meta/src/hummock/manager/mod.rs

```diff
@@ -2024,6 +2026,15 @@ where
             );
     }
 
+    fn notify_stats(&self, stats: &HummockVersionStats) {
+        self.env
+            .notification_manager()
+            .notify_frontend_without_version(Operation::Update, Info::HummockStats(stats.clone()));
+        self.env
+            .notification_manager()
+            .notify_compute_without_version(Operation::Update, Info::HummockStats(stats.clone()));
```

Comment:
I thought the plan is to execute query related to version stats in frontend. Do we still need to notify compute node in this case?

Reply:
Yes, indeed, but I wanted to get the draft PR first and then go through and remove the Compute node work in parallel to people making comments :).